crates/accelerate/src/target_transpiler/mod.rs

```diff
@@ -428,9 +428,14 @@ impl Target {
         py: Python<'py>,
         instruction: &str,
     ) -> PyResult<Bound<'py, PyAny>> {
-        match self._operation_from_name(instruction) {
-            Ok(instruction) => instruction.into_pyobject(py),
-            Err(e) => Err(PyKeyError::new_err(e.message)),
+        match self.operation_from_name(instruction) {
+            Some(TargetOperation::Normal(operation)) => {
+                operation.into_pyobject(py).map(|x| x.to_owned())
+            }
+            Some(TargetOperation::Variadic(ob)) => Ok(ob.bind(py).clone()),
+            None => Err(PyKeyError::new_err(format!(
+                "Instruction {instruction} not in target"
+            ))),
```

Comment:
This conversion here is fine. I did think the `TargetOperation` struct implemented `IntoPyObject` and would do exactly this but instead it implements it via `IntoPyObjectRef` which I don't believe does the same thing.

Reply:
Oh, I can change it that - I didn't spot it, and I originally had something different written here (I was thrown off by the Rust-space and Python-space versions handling variadics differently).  Done in 98cbe6e0d8.